interfaces/adfsuite/adf.py

```diff
@@ -69,13 +69,13 @@ def get_dipole_vector(self, unit='au'):
         raise ResultsError("'Dipole' not present in 'Properties' section of {}".format(self._kfpath()))
 
 
-    def get_gradients(self, eUnit='au', lUnit='bohr'):
-        """get_gradients(eUnit='au', lUnit='bohr')
+    def get_gradients(self, energy_unit='au', dist_unit='bohr'):
```

Comment:
Note that this name change _will_ break any existing (user) code relying on the old keyword arguments.

Reply:
Yeah it does... I guess the big question I have here: Will SCM remove this interface soon anyway or is it gonna stay and make it worth my while to work on it?

Will make a better commit that does not break compatibility for now.

Reply:
Hacky fix, let me know what you think. I really don't like it if each interface uses its own keywords, they should be standardized... But not sure how to enforce that for each interface. Perhaps write it down somewhere?